hack/run-functests.sh

```diff
@@ -39,6 +39,15 @@ else
   EXEC_TESTS="cnf-tests/test-run.sh -ginkgo.focus=$FOCUS -junit $TESTS_REPORTS_PATH -report $TESTS_REPORTS_PATH"
 fi
 
+reports="cnftests_failure_report.log setup_failure_report.log validation_failure_report.log"
+
+for report in $reports; do
+  if [[ -f "$TESTS_REPORTS_PATH/$report" ]]; then    
+    gzip "$TESTS_REPORTS_PATH/$report"
```

Comment:
do we have gzip inside the cnf container?

Reply:
That is happening outside the container (the user's terminal OR the pod that is running CI tests).
On CI side, will find out :P

Reply:
Just ssh-ed into the test runner pod, it has it onboard